zipline/finance/performance/tracker.py

```diff
@@ -69,6 +69,8 @@
 from zipline.finance import trading
 from . period import PerformancePeriod
 
+trd_env = trading.TradingEnvironment()
```

Comment:
Don't do this at module level, it causes all kinds of craziness, I would look for usages of the `with_environment decorator from the trading module.